### DISCUSSION TITLE
French translation + suggestion for internal mode

### DISCUSSION THREAD

**Discussion by joachimBrindeau** [kromitgmbh/titra#80] [Ideas]
Hello,

Please find [**here** ](https://we.tl/t-6EA0JJWUEw) the complete french translation.
Hope it helps the project.

By the way, it would be great to have an "internal mode" for companies who are not service providers.
A simple switch that would hide everything related to clients, rates and invoices.

Cheers!

**Comment by faburem** [comment]
thanks, that's awesome - will integrate it soon!

**Comment by faburem** [comment]
French translation has shipped in titra version 0.57.0! Thanks again for your amazing support of this project. We will add more settings to enable/disable certain fields. An internal mode would be a good fit for our new extension feature. I will keep you posted here if there is any news on this part.
